src/reuse/lint.py

```diff
@@ -264,7 +267,9 @@ def custom_serializer(obj: Any) -> Any:
     )
 
 
-def format_lines(report: ProjectReport) -> str:
+def get_errors(
+    report: ProjectReport,
+) -> Iterable[tuple[Path | str | None, str]]:
```

Comment:
The tuple should probably be a NamedTuple

Reply:
Also, I don't think `Iterable` is correct. Generators in Python are annoying to type correctly. I think it's `Iterator[type, False, False]` or some such. I'll look this up.